Gemfile.lock

```diff
@@ -73,82 +73,34 @@ GIT
       script_utils (= 0.0.4)
 
 PATH
-  remote: modules/appeals_api
+  remote: modules
   specs:
     appeals_api (0.0.1)
       sidekiq
-
-PATH
-  remote: modules/apps_api
-  specs:
     apps_api (0.0.1)
       faraday
       sidekiq
-
-PATH
-  remote: modules/claims_api
-  specs:
     claims_api (0.0.1)
-
-PATH
-  remote: modules/covid_research
-  specs:
     covid_research (0.1.0)
       rails (~> 6.0.3, >= 6.0.3.2)
       sidekiq
-
-PATH
-  remote: modules/health_quest
-  specs:
     health_quest (0.1.0)
-
-PATH
-  remote: modules/mobile
-  specs:
     mobile (0.1.0)
-
-PATH
-  remote: modules/openid_auth
-  specs:
     openid_auth (0.0.1)
-
-PATH
-  remote: modules/va_forms
-  specs:
     va_forms (0.0.1)
       faraday
       nokogiri
       paper_trail
       sidekiq
-
-PATH
-  remote: modules/vaos
-  specs:
     vaos (0.1.0)
       sidekiq
-
-PATH
-  remote: modules/vba_documents
-  specs:
     vba_documents (0.0.1)
       aws-sdk-s3 (~> 1)
       sidekiq
-
-PATH
-  remote: modules/veteran_confirmation
-  specs:
+    veteran (0.0.1)
```

Comment:
I'm thinking this was moved for alphabetizing purposes? Correct?

Reply:
Yeah. The diff looked a little weird here but this was all bundlers doing /shrug